internal/testrunner/runners/pipeline/runner.go

```diff
@@ -321,8 +354,13 @@ func (r *runner) runTestCase(ctx context.Context, testCaseFile string, dsPath st
 		return tr, nil
 	}
 
-	if r.options.WithCoverage {
-		tr.Coverage, err = GetPipelineCoverage(r.options, r.pipelines)
+	if r.withCoverage {
+		tr.Coverage, err = GetPipelineCoverage(PipelineRunnerOptions{
```

Comment:
Nit. Is this method used in other packages?
```suggestion
		tr.Coverage, err = getPipelineCoverage(PipelineRunnerOptions{
```

Reply:
I'll check it, and if it is not used in other packages I'll update it  :+1:

Reply:
Done in https://github.com/elastic/elastic-package/pull/1888/commits/51b825de892fcc46fde39ef0c1bc5d09e8297c3b